src/Angor/Shared/Utilities/NostrConversionHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Nostr.Client.Utils;  
+using System;
+
+namespace Angor.Shared.Utilities
+{
+    public static class NostrConversionHelper
+    {
+        /// <summary>
+        /// Convert Bech32 key to hex
+        /// </summary>
+        public static string? ConvertBech32ToHex(string bech32Key)
+        {
+            try
+            {
+                string? hrp;
+                return NostrConverter.ToHex(bech32Key, out hrp);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error converting Bech32 to hex: {ex.Message}");
```

Comment:
We don't use console we use a logger class, as this is a static class so just throw the exception up

Reply:
ok fine with me